src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,17 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
-        return null;
+        if (storageDao.get(user.getLogin()) != null) {
+            throw new RuntimeException("Duplicate login");
```

Comment:
```suggestion
            throw new RuntimeException("Duplicate login " + user.getLogin());
```

It's better to make exception messages more informative